### PR TITLE
Fix Automation auth issue - External Data Connector not working

### PR DESCRIPTION
**Pull request by PClmnt**
## Description
Seems to be an auth issue introduced by the recent rest request changes.  

I'm not sure this is the most graceful way to handle this error, I have tested and it does fix the issue but i'm just trying to understand if we actually need the `getAuthConfig` call when running an automation?

Addresses: 
#6611

### CODE REVIEW

File: packages/server/src/api/controllers/query/index.ts [hunk 1]
```diff
@@ -214,11 +219,11 @@ async function execute(ctx: any, opts = { rowsOnly: false }) {
 }
 
 export async function executeV1(ctx: any) {
-  return execute(ctx, { rowsOnly: true })
+  return execute(ctx, { rowsOnly: true, isAutomation: false })
 }
 
-export async function executeV2(ctx: any) {
-  return execute(ctx, { rowsOnly: false })
+export async function executeV2(ctx: any, isAutomation?: any) {
```
Comment (mike12345567) [inline packages/server/src/api/controllers/query/index.ts:225]: Preferably all boolean parameters should be wrapped in objects, options objects like `ctx: any, { isAutomation }: { isAutomation?: boolean } = {}`

Avoids situations with calls like `executeV2(ctx, false)` - difficult to see what this does vs `executeV2(ctx, { isAutomation: false })`

**Review by mike12345567 — APPROVED** [review]
LGTM, this is probably the best way to handle this anyway, other than adding it as a property to the `ctx` e.g. `ctx.automation = true`. I'd say this is a slightly larger refactor however, as I would wrap all controller calls from automations with a function that applies this parameter to the `ctx`, making it a consistent and dependable parameter that any controller could use.

**Comment by deanhannigan** [comment]
Hey @PClmnt, yes at the minute the automations have no need for access to the authentication information. This is currently only used for rest requests and discerning which OIDC config to use when refreshing an access token.

**Review by deanhannigan — APPROVED** [review]
LGTM, great catch!

**Pull request merged**